Cargo.toml

```diff
@@ -60,16 +56,21 @@ flate2 = "1"
 fs_at.workspace = true
 git-testament = "0.2"
 home = "0.5.4"
-lazy_static.workspace = true
 libc = "0.2"
-once_cell = { workspace = true, optional = true }
+once_cell.workspace = true
 opener = "0.6.0"
+# Used by `curl` or `reqwest` backend although it isn't imported by rustup:
```

Comment:
I see what you mean here, but I'm afraid for someone without any context of this change, these comments might seem to `openssl` and some other stuff below.

The following should be able to limit the scope of this comment to `openssl`:

```suggestion
# `openssl` is used by `curl` or `reqwest` backend although it isn't imported by rustup:
```

Reply:
Fair, fixed as suggested!